worker/src/main/java/gov/cms/ab2d/worker/processor/JobPreProcessorImpl.java

```diff
@@ -57,26 +61,28 @@ public Job preprocess(String jobUuid) {
             throw new IllegalArgumentException(errMsg);
         }
 
-        Optional sinceValue = Optional.ofNullable(job.getSince());
+        Optional<Contract> contractOptional = contractRepository.findContractByContractNumber(job.getContractNumber());
```

Comment:
Shouldn't we throw an error if the contract can't be found? Seems like a pretty big issue if the contract number for a job doesn't match an existing contract. In fact, we won't be able to run anything if that is the case. I would just thrown an exception here and fail the job outright.